contrib/cirrus/runner.sh

```diff
@@ -233,6 +233,17 @@ function _run_consistency() {
     SUGGESTION="run 'make generate-bindings' and commit all changes" ./hack/tree_status.sh
     make completions
     SUGGESTION="run 'make completions' and commit all changes" ./hack/tree_status.sh
+    if  [[ -z "$CIRRUS_TAG" ]] && \
+        req_env_vars CIRRUS_CHANGE_IN_REPO CIRRUS_PR DEST_BRANCH &> /dev/null
+    then
+        local base diffs
+        base=$(git merge-base $DEST_BRANCH $CIRRUS_CHANGE_IN_REPO)
+        diffs=$(git diff $base $CIRRUS_CHANGE_IN_REPO)
+        if egrep -q '^\+.+io/ioutils' || egrep -q '^\+.+ioutils\..+'; then
```

Comment:
I suspect that this test is going to trigger on this very PR :wink:

Reply:
Update: it's going to hang, actually, because there's no argument to grep. Anyhow, this needs to be fixed: to grep only `.go` files, to ignore `vendor`, and probably more. With time - not today.

Reply:
damn.

Reply:
Actually we want it to fail on `.go` and `.vendor`, they will also fail to compile under go 1.19.